menu_view.cpp

```diff
@@ -1,6 +1,7 @@
 #include "menu_view.h"
 #include "coordinate.h"
 
+#ifndef LOGIC_ONLY // that is, compiled on GitHub Actions

```

Comment:
I could not have predicted this `ifndef` would be a good idea, but I assume you ran into build problems and found out that this was a solution. Well done!

Reply:
No, this is let over from an attempt at adding a test block to `menu_view`. I have no idea what this is actually for, I'll just rm it